<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>550</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>144</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="emailLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>17</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>142</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="buddy">
          <cstring>emailInput</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="emailInput"/>
       </item>
       <item>
        <widget class="QLabel" name="passLabel">
         <property name="text">
          <string>Password</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>passInput</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passInput"/>
       </item>
       <item>
        <widget class="QPushButton" name="loginBtn">
         <property name="text">
          <string>Login</string>
         </property>
         <property name="shortcut">
          <string>Return</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>emailInput</zorder>
      <zorder>passInput</zorder>
      <zorder>passLabel</zorder>
      <zorder>emailLabel</zorder>
      <zorder>loginBtn</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="logOutAction"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <action name="logOutAction">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
